### PR TITLE
docs(ThreadManager): Fix typo

### PR DESCRIPTION
**Pull request by Jiralite**
**Please describe the changes this PR makes and why it should be merged:**
Fixes a typo. Also tweaked the wording to refer to the guild's features as this is more accurate.

**Status and versioning classification:**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

File: src/managers/ThreadManager.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ class ThreadManager extends CachedManager {
    * * `1440` (1 day)
    * * `4320` (3 days) <warn>This is only available when the guild has the `THREE_DAY_THREAD_ARCHIVE` feature.</warn>
    * * `10080` (7 days) <warn>This is only available when the guild has the `SEVEN_DAY_THREAD_ARCHIVE` feature.</warn>
-   * * `'MAX'` Based on the guilds boost count
+   * * `'MAX'` Based on the guild's features
```
Comment (ImRodry) [inline src/managers/ThreadManager.js:69]: Why remove this
```suggestion
   * * `'MAX'` Based on the guild's boost level
```

Comment (ckohen) [reply src/managers/ThreadManager.js:69]: It's not *necessarily* the boost count that grants features. They are usually tied, but some guilds may have this feature with even 0 boosts.

Comment (ImRodry) [reply src/managers/ThreadManager.js:69]: Yeah only if they’re partnered or verified, in which case boost level 3 is implicit even with no boosts

Comment (ajpalkovic) [reply src/managers/ThreadManager.js:69]: I wouldn't say that is necessarily accurate

the guild feature will exist for private threads if they have private threads, same for 3/7 day times

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**